subcommand/cert-manager/command.go

```diff
@@ -0,0 +1,243 @@
+package certmanager
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"errors"
+	"flag"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/hashicorp/consul-k8s/helper/cert"
+	"github.com/hashicorp/consul-k8s/subcommand/flags"
+	"github.com/mitchellh/cli"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type Command struct {
+	flagSet   *flag.FlagSet
+	httpFlags *flags.HTTPFlags
+	UI        cli.Ui
+
+	flagWebhookName     string // MutatingWebhookConfiguration for updating
+	flagAutoHosts       string // SANs for the auto-generated TLS cert.
+	flagSecretName      string // Name of secret where certificates will be written to.
+	flagSecretNamespace string // Namespace of the secret where certificates will be written to.
+	flagCertFile        string // TLS cert for listening (PEM)
+	flagKeyFile         string // TLS cert private key (PEM)
+
+	clientset kubernetes.Interface
+
+	once sync.Once
+	help string
+}
+
+func (c *Command) init() {
+	c.flagSet = flag.NewFlagSet("", flag.ContinueOnError)
+	c.flagSet.StringVar(&c.flagWebhookName, "webhook-name", "",
+		"MutatingWebhookConfiguration name. If specified, will auto generate cert bundle.")
```

Comment:
I thought it was the existence of `-tls-cert-file` and `-tls-key-file` that controls whether we auto generate the cert bundle?

Reply:
The cert file and key file are files on disk and if they are present, we use those and do not generate the bundle. But we should get rid of this behavior for now